_data/sidebars/apim_3_x_sidebar.yml

```diff
@@ -77,19 +77,25 @@ entries:
                   - title: Introduction
                     output: web
                     url: /apim/3.x/apim_installation_guide_docker_introduction.html
-                  - title: Images
+                  - title: Quick install with Docker Compose
                     output: web
-                    url: /apim/3.x/apim_installation_guide_docker_images.html
-                  - title: Images (EE)
+                    url: /apim/3.x/apim_installation_guide_docker_compose_quickstart.html
+                  - title: Quick install with Docker Compose (EE)
                     output: web
-                    url: /apim/3.x/apim_installation_guide_docker_images_ee.html
-                  - title: Compose
+                    url: /apim/3.x/apim_installation_guide_docker_compose_quickstart_ee.html
+                  - title: Custom Install with Docker Compose
                     output: web
                     url: /apim/3.x/apim_installation_guide_docker_compose.html
-                  - title: Compose (EE)
+                  - title: Custom Install with Docker Compose (EE)
                     output: web
                     url: /apim/3.x/apim_installation_guide_docker_compose_ee.html
-                  - title: Customize
+                  - title: Docker Images Install
+                    output: web
+                    url: /apim/3.x/apim_installation_guide_docker_images.html
+                  - title: Docker Images Install (EE)
+                    output: web
+                    url: /apim/3.x/apim_installation_guide_docker_images_ee.html
+                  - title: Further Customization
```

Comment:
Consider adding "Further Customization as a subheader under the custom install options